docs/contributing/writing-instrumentation.md

```diff
@@ -113,6 +113,34 @@ the base class you wrote earlier, but in this, create a client using none of the
 only the ones offered by the library. Implement the `AgentTestRunner` trait for common setup logic,
 and try running. All the tests should pass for agent instrumentation too.
 
+Note that all the tests inside the `javaagent` module will be run using the shaded `-javaagent`
```

Comment:
Does word `shaded` here mean anything? I mean, is this agent used for tests somehow different from the "usual" agent that we release?

Reply:
I updated it to "fully shaded", just want to be clear that shading takes place, since that is an extra layer of safety that this approach brings